test/unit/ngsiv2/examples/contextRequests/updateContextMultientityPlugin2.json

```diff
@@ -0,0 +1,22 @@
+{
+  "actionType": "UPDATE",
+  "entities": [
+    {
+      "id": "ws4",
+      "type": "WeatherStation",
+      "pressure": {
+        "type": "Hgmm",
+        "value": "52"
+      }
+    },
+    {
+      "humidity": {
+        "type": "Percentage",
+        "value": "12"
+      },
+      "type": "WeatherStation",
+      "id": "Higro2000"
+    }
+  ]
+}
+
```

Comment:
This file seems identical to updateContextMultientityPlugin1.json or at least I'm unable to spot the difference...

Reply:
I finally find the diference: entity type (here is "WeatherStation" and there is "Higrometer").

NTC. Sorry for the noise ;)